src/components/Capture/capture.js

```diff
@@ -193,25 +194,46 @@ class Capture extends Component {
     }
   }
 
-  onApiError = (error) => {
-    this.setState({error, uploadInProgress: false})
+  onfidoErrorFieldMap = ([key, val]) => {
```

Comment:
What's the reason for bringing this logic into Capture? I don't see how it helps us to track screens